server/controllers/request.js

```diff
@@ -0,0 +1,77 @@
+// require Request Model
+const Request = {};
+
+exports.getRequestById = (req, res, next, id) => {
+  Request
+    .findById(id)
+    .exec((error, request) => {
+      if (error) {
+        return res.status(400).json({
+          error: "No request was found in DB"
+        })
+      }
+
+      req.request = request;
+      next();
+
+    })
+}
+
+exports.getAllRequest = (req, res) => {

```

Comment:
I'm not sure I see a use case for this route, would it make more sense to get all requests for the logged in user

Reply:
I think `limit` is not required in this case, but we can sort requests by order in which they are created. Let me know about it.